embassy-stm32/build.rs

```diff
@@ -1434,39 +1434,53 @@ fn main() {
 
         if is_rcc_name(e.name) {
             let enum_name = format_ident!("{}", e.name);
-            let mut muls = Vec::new();
-            let mut divs = Vec::new();
+            let muls_ident = format_ident!("{}_MULS", e.name.to_uppercase());
+            let divs_ident = format_ident!("{}_DIVS", e.name.to_uppercase());
+
+            let mut mul_values = Vec::new();
+            let mut div_values = Vec::new();
+            let mut min_val = 0;
             for v in e.variants {
+                // check if the diff is greater than 1, if so, we need to fill in rserverd values
```

Comment:
this is assuming `e.variants` is sorted increasing by value, which doesn't have to be the case.